index.html

```diff
@@ -6,10 +6,18 @@
       </title>  
     </head>
     <body>
-        <h1>Welcome</h1>
-        <p>This is my first page.</p>
+        <h1 id="headerTitle">Welcome</h1>
+        <p>This is my first page.</p> <br><br>
```

Comment:
Lønner seg å fikse mellomrom med CSS i stedet for br-tagger

Reply:
Ok. Skal jobbe med saken. Venter med å merge.

Reply:
Neida, det er greit! Bare å merge!